content/en/docs/contribute/start.md

```diff
@@ -247,8 +247,10 @@ pull request if it detects that you pushed a new branch to your fork.
     is the same as the commit summary, but you can change it if needed. The
     body is populated by your extended commit message (if present) and some
     template text. Read the template text and fill out the details it asks for,
-    then delete the extra template text. Leave the
-    **Allow edits from maintainers** checkbox selected. Click
+    then delete the extra template text. If you add to the description `fixes #<000000>` 
+    or `closes #<000000>`, where `#<000000>` is the issue number,
```

Comment:
s/is the issue number/is the number of an associated issue

Reply:
done!